servicetalk-loadbalancer/src/main/java/io/servicetalk/loadbalancer/RoundRobinLoadBalancerFactory.java

```diff
@@ -49,11 +49,18 @@
  * suggests otherwise. This can lead to situations where connections will be used to their maximum capacity
  * (for example in the context of pipelining) before new connections are created.</li>
  * <li>Closed connections are automatically pruned.</li>
- * <li>If {@link #eagerConnectionShutdown} is set to {@code true}, connections are immediately closed
- * for an {@link ServiceDiscovererEvent#isAvailable() unavailable} address. When {@code false} is used, connections
- * to addresses marked as {@link ServiceDiscovererEvent#isAvailable() unavailable} are used for requests,
- * but no new connections are created for them. In case the address' connections are busy, another host is tried.
- * If all hosts are busy, selection fails with a {@link io.servicetalk.client.api.ConnectionRejectedException}.</li>
+ * <li>When {@link Publisher}&lt;{@link ServiceDiscovererEvent}&gt; delivers events with
```

Comment:
Thanks for the updated javadoc here. Would be nice to provide a short summary what statuses are supported by this implementation, before you go into "The created instances have the following behaviour" details.